hack/test-cmd.sh

```diff
@@ -202,11 +208,31 @@ kube::util::wait_for_url "http://127.0.0.1:${API_PORT}/healthz" "apiserver"
 kube::log::status "Starting controller-manager"
 "${KUBE_OUTPUT_HOSTBIN}/kube-controller-manager" \
   --port="${CTLRMGR_PORT}" \
+  --kube-api-content-type="${KUBE_TEST_API_TYPE-}" \
   --master="127.0.0.1:${API_PORT}" 1>&2 &
 CTLRMGR_PID=$!
 
 kube::util::wait_for_url "http://127.0.0.1:${CTLRMGR_PORT}/healthz" "controller-manager"
-kube::util::wait_for_url "http://127.0.0.1:${API_PORT}/api/v1/nodes/127.0.0.1" "apiserver(nodes)"
+
+if [[ "$(go env GOHOSTOS)" == "linux" ]]; then
```

Comment:
It's been split to its own PR

On Mon, Apr 25, 2016 at 7:26 PM, Daniel Smith <notifications@github.com>
wrote:

> In hack/test-cmd.sh
> <https://github.com/kubernetes/kubernetes/pull/24710#discussion_r61007375>
> :
>
> >    --master="127.0.0.1:${API_PORT}" 1>&2 &
> >  CTLRMGR_PID=$!
> >
> >  kube::util::wait_for_url "http://127.0.0.1:${CTLRMGR_PORT}/healthz" "controller-manager"
> > -kube::util::wait_for_url "http://127.0.0.1:${API_PORT}/api/v1/nodes/127.0.0.1" "apiserver(nodes)"
> > +
> > +if [[ "$(go env GOHOSTOS)" == "linux" ]]; then
>
> also unrelated?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/24710/files/e4a1f3c4b34ba5ef0c8f08009e202d0b048498ff#r61007375>
>